devops_sccs/plugins/bitbucketcloud.py

```diff
@@ -120,18 +120,19 @@ async def __handle_push(self,UUID,responseJson):
                     env.environment = self.cd_environments[index]["name"]
                     env.buildstatus = str(commit_status_state.INPROGRESS if (newName == "master") else commit_status_state.SUCCESSFUL )
                     i = 0
+                    repoName=responseJson["repository"]["name"]
```

Comment:
It can be newName instead , since it's the name of  the branch where it will update.